ui/packages/catalog-ui-search/src/main/webapp/js/Common.js

```diff
@@ -54,7 +54,7 @@ module.exports = {
   //randomly generated guid guaranteed to be unique ;)
   undefined: '2686dcb5-7578-4957-974d-aaa9289cd2f0',
   coreTransitionTime: 250,
-  generateUUID: function() {
+  generateUUID(properties = require('properties')) {
```

Comment:
This is an interesting solution.  I'd recommend we break out the generateUUID function into it's own module (now or in the future) since it's the only thing in Common.js that depends on business logic (properties.js) specific to our application.